packages/lab/src/avatar/Avatar.css

```diff
@@ -24,15 +27,12 @@
   overflow: hidden;
   user-select: none;
   border-radius: 50%;
+  outline: auto;
```

Comment:
this gives a white outline in dark mode, which I don't think is intended?

Reply:
this was to avoid a blue outline otherwise, might have to look into it a bit more